doc/nrf/conf.py

```diff
@@ -112,9 +113,18 @@
 # built documents.
 #
 # The short X.Y version.
-version = '1.3.99'
+version = 'unknown'
+
 # The full version, including alpha/beta/rc tags.
-release = '1.3.99'
+release = 'unknown'
+
+with open(os.path.join(NRF_BASE, 'VERSION')) as version_file:
+    for line in version_file:
+        match = re.search(r'^(\d*\.\d*\.\d*)(-[^ \r\n\t]*)?', line)
```

Comment:
I believe the second match could be `(-\S*)?`

Reply:
Thanks for the suggestion.
Done.